taggit/models.py

```diff
@@ -101,11 +102,15 @@ class Meta:
 
     @classmethod
     def tag_model(cls):
-        return cls._meta.get_field_by_name("tag")[0].rel.to
+        if VERSION < (1, 8):
+            return cls._meta.get_field_by_name('tag')[0].rel.to
+        return cls._meta.get_field('tag').rel.to
```

Comment:
Once the other function gets moved to `utils.py`, can you utilize it here so we can reduce the number of places we do version checking